src/framework/mlt_playlist.c

```diff
@@ -218,6 +218,13 @@ static int mlt_playlist_virtual_refresh( mlt_playlist self )
 		frame_count += self->list[ i ]->frame_count;
 	}
 
+	// force loop
+	if ( mlt_properties_get( properties, "force_loop" ) )
```

Comment:
This feature require to implement nested playlist loop.

Reply:
Explain more about "nested playlist." Maybe a workaround is to use a tractor that has eof=loop, and tractor contains the inner playlist.

Reply:
i am trying to play such playlist https://gist.github.com/max-verem/e29ea01d510bfa7b0cb3c39f76759814 but it does not play as i expect. There are no *virtual* duration for nested playlist with loop feature.

so, to avoid previous functionality breaking, i am introducing a new feature that can be used in a playlist: https://gist.github.com/max-verem/eae8a22f6c57315bdf77c218cab11dd0

hope it is clear thay i want to implement with this patch? may be there are other way to name it and provide *virtually* unlimited length loop...

Reply:
> This feature require to implement nested playlist loop.

Hi, sorry to ask this here, but by nested playlist you mean a nested timeline? If so this is a very important feature! :)